l10n_it_fatturapa_sale/models/sale_order_line.py

```diff
@@ -0,0 +1,54 @@
+#  Copyright 2020 Simone Rubino - Agile Business Group
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class SaleOrderLine (models.Model):
+    _inherit = 'sale.order.line'
+
+    related_documents = fields.One2many(
+        comodel_name='fatturapa.related_document_type',
+        inverse_name='sale_order_line_id',
+        string='Related Documents',
+        copy=False,
+    )
+    admin_ref = fields.Char(
+        string="Admin. ref.",
+        size=20,
+        copy=False,
+    )
+
+    @api.multi
+    def _prepare_invoice_line(self, qty):
+        self.ensure_one()
+        invoice_line_vals = super()._prepare_invoice_line(qty)
+        sale_line_documents = self.related_documents
```

Comment:
```suggestion
        sale_line_documents = self.related_documents
        invoice_line_vals.update({
            'related_documents': self.related_documents,
            'admin_ref': self.admin_ref,})
        return invoice_line_vals
```

Reply:
In questo modo si sovrascrivono eventuali valori già presenti in `invoice_line_vals`, ma suppongo sia un rischio accettabile.

Reply:
@tafaRU se secondo te è un problema bloccante per approvare la PR mergio la modifica, altrimenti se è solo una questione di gusti lascerei l'implementazione attuale.